src/imports/logger/ErrorsTransport.js

```diff
@@ -0,0 +1,77 @@
+//import Rollbar from 'rollbar'
+
+import * as Sentry from '@sentry/node'
+import Transport from 'winston-transport'
+import { SPLAT } from 'triple-beam'
+import { forEach } from 'lodash'
+import Config from '../../server/server.config'
+
+const { env, sentryDSN, version, network } = Config
+const logEnvAllowed = !['test', 'development'].includes(env)
+
+// let rollbar
+// if (logEnvAllowed && rollbarToken) {
+//   rollbar = new Rollbar({
+//     accessToken: rollbarToken,
+//     captureUncaught: true,
+//     captureUnhandledRejections: true,
+//     payload: {
+//       environment: process.env.NODE_ENV
+//     }
+//   })
+// }
+
+let sentryInitialized = false
```

Comment:
1. Inject Sentry & Config to the CustomTransport constructor
2. move this to the constructor
3. add factory method which will pass imported Config & Sentry in addition to the opts

e.g.
```javascript
class CustomTransport extends Transport {
   sentryInitialized = false

   static factory = options => new CustomTransport(options, Config, Sentry)

   // eslint-disable-next-line
   constructor(opts, Config, Sentry) {
     const { env, sentryDSN, version, network } = Config
     
     super(opts)

     // TODO initialize Sentry
    this.sentryInitialized = true
   }
```